examples/demo_interface.cpp

```diff
@@ -18,12 +18,6 @@
 
 using namespace simulator;
 
-DECLARE_int32(context);
-DEFINE_string(game, "simple_game", "name of game");
-DEFINE_int32(num_agents, 1, "number of agents");
-DEFINE_int32(port, 50000, "port number for socket communication");
-DEFINE_int32(client_id, -1, "client id; -1 for server");
-
```

Comment:
I think we'd better keep flags_game and flags_port, so that users can specify the game and port without changing the code.

Reply:
This is only for demo. FLAGS_port should be add into game_player but not here.

Reply:
let users to choose port when running this demo. we never know when pre-selected port will be used by other processes.